test/ServerInvoiceTest.php

```diff
@@ -69,26 +74,26 @@ function testBasic()
 
         # Retrieve each invoice by id
         $invoice = new Invoice();
-        $invoice->invoiceId =  $invoices[0]->invoiceId;
+        $invoice->setInvoiceId($invoices[0]->getInvoiceId());
         $invoice = $invoice->get(false, $this->tenant->getTenantHeaders());
         $this->assertNotEmpty($invoice);
-        $this->assertNotEmpty($invoice->accountId);
-        $this->assertNotEmpty($invoice->invoiceId);
-        $this->assertNotEmpty($invoice->currency);
-        $this->assertEquals($invoice->amount, 0);
-        $this->assertEquals($invoice->balance, 0);
-        $this->assertEmpty($invoice->items);
+        $this->assertNotEmpty($invoice->getAccountId());
+        $this->assertNotEmpty($invoice->getInvoiceId());
+        $this->assertNotEmpty($invoice->getCurrency());
+        $this->assertEquals($invoice->getAmount(), 0);
+        $this->assertEquals($invoice->getBalance(), 0);
+        $this->assertEmpty($invoice->getItems());
 
         $invoice = new Invoice();
-        $invoice->invoiceId =  $invoices[1]->invoiceId;
+        $invoice->setInvoiceId($invoices[1]->getInvoiceId());
         $invoice = $invoice->get(true, $this->tenant->getTenantHeaders());
         $this->assertNotEmpty($invoice);
-        $this->assertNotEmpty($invoice->accountId);
-        $this->assertNotEmpty($invoice->invoiceId);
-        $this->assertNotEmpty($invoice->currency);
-        $this->assertEquals($invoice->amount, 500);
-        $this->assertEquals($invoice->balance, 0);
-        $this->assertNotEmpty($invoice->items);
-        $this->assertEquals(1, count($invoice->items));
+        $this->assertNotEmpty($invoice->getAccountId());
+        $this->assertNotEmpty($invoice->getInvoiceId());
+        $this->assertNotEmpty($invoice->getCurrency());
+        $this->assertEquals($invoice->getAmount(), 500);
```

Comment:
The account's currency is indeed in USD: 

```
MariaDB [killbill]> select * from accounts\G;
*************************** 1. row ***************************
                     record_id: 339
                            id: 2d1b6abb-2415-4156-a7a5-6db9d5c3e4c8
                  external_key: 58e3514225924
                         email: test-58e3514225924@kill-bill.org
                          name: Killbill php test
             first_name_length: 4
                      currency: USD
       billing_cycle_day_local: 12
             parent_account_id: NULL
is_payment_delegated_to_parent: NULL
             payment_method_id: 021d76ad-1d43-477c-97e7-67ec22854140
                     time_zone: UTC
                        locale: NULL
                      address1: 12 rue des ecoles
                      address2: Poitier
                  company_name: Renault
                          city: NULL
             state_or_province: Poitou
                       country: France
                   postal_code: NULL
                         phone: 81 53 26 56
                         notes: NULL
                      migrated: NULL
      is_notified_for_invoices: 0
                  created_date: 2017-04-03 18:31:22
                    created_by: phpTester
                  updated_date: 2017-04-03 18:31:22
                    updated_by: phpTester
              tenant_record_id: 461
1 row in set (0.00 sec)
```

These are the entries in the `invoice_items` table:
```
MariaDB [killbill]> select * from invoice_items\G;
*************************** 1. row ***************************
        record_id: 230
               id: f3484c58-117b-43e3-ad61-b120f022cfa5
             type: FIXED
       invoice_id: 28c3f17c-2788-4e0f-be64-b7a63ab99a17
       account_id: 2d1b6abb-2415-4156-a7a5-6db9d5c3e4c8
 child_account_id: NULL
        bundle_id: 2fd1e695-f2d7-44c0-8b6a-45312a9418aa
  subscription_id: 21be3d8e-7267-4271-b376-84b9a4bf65b8
      description: sports-monthly-trial
        plan_name: sports-monthly
       phase_name: sports-monthly-trial
       usage_name: NULL
       start_date: 2017-04-03
         end_date: NULL
           amount: 0.000000000
             rate: NULL
         currency: USD
   linked_item_id: NULL
       created_by: SubscriptionBaseTransition
     created_date: 2017-04-03 18:31:23
account_record_id: 339
 tenant_record_id: 461
*************************** 2. row ***************************
        record_id: 231
               id: 42e482b8-c5eb-4bdf-a2e1-84ac1011e77d
             type: RECURRING
       invoice_id: c3bd7806-83d2-473e-9f13-2e14e4605b81
       account_id: 2d1b6abb-2415-4156-a7a5-6db9d5c3e4c8
 child_account_id: NULL
        bundle_id: 2fd1e695-f2d7-44c0-8b6a-45312a9418aa
  subscription_id: 21be3d8e-7267-4271-b376-84b9a4bf65b8
      description: sports-monthly-evergreen
        plan_name: sports-monthly
       phase_name: sports-monthly-evergreen
       usage_name: NULL
       start_date: 2017-05-03
         end_date: 2017-05-12
           amount: 150.000000000
             rate: 500.000000000
         currency: USD
   linked_item_id: NULL
       created_by: SubscriptionBaseTransition
     created_date: 2017-05-04 18:31:26
account_record_id: 339
 tenant_record_id: 461
2 rows in set (0.00 sec)
```